pyproject.toml

```diff
@@ -21,6 +21,7 @@ include = ["feedparser/py.typed"]
 [tool.poetry.dependencies]
 python = ">=3.7"
 sgmllib3k = "^1.0.0"
+requests = ">=2.20.0"
```

Comment:
It made `requests` a hard dependency, which may not be good news for those using `feedparser` as a "pure" parser.